tests/core/core-configuration-layer-utest.el

```diff
@@ -1044,16 +1044,18 @@
         (layer-packages '(pkg1 pkg2 pkg3))
         (mocker-mock-default-record-cls 'mocker-stub-record))
     (mocker-let
-     ((file-exists-p (f) ((:output t :occur 2)))
-      (load (f &optional noerr nomsg) ((:output nil :occur 1))))
-     (should (equal (cfgl-layer "layer"
-                                :name 'layer
-                                :disabled-for nil
-                                :variables nil
-                                :packages '(pkg1 pkg2 pkg3)
-                                :selected-packages 'all
-                                :dir spacemacs-start-directory)
-                    (configuration-layer/make-layer 'layer layer 'used))))))
+        ((locate-file
```

Comment:
minor issue: indentation level isn't right. (for this and other tests)

Reply:
@lebensterben  fixed, please review again. Thanks